models/articles.models.js

```diff
@@ -21,13 +22,39 @@ exports.selectArticleById = (articleId) => {
     });
 };
 
-exports.selectAllArticles = () => {
-    const queryString = `SELECT articles.author, articles.title, articles.article_id, articles.topic, articles.created_at, articles.votes, articles.article_img_url, COUNT(comments.comment_id) AS comment_count
-    FROM articles
-    LEFT JOIN comments
-    ON articles.article_id = comments.article_id
-    GROUP BY articles.article_id
-    ORDER BY articles.created_at DESC;`;
+exports.selectAllArticles = (topic, sortBy = "created_at", order = "desc") => {
+    const validTopics = ["mitch", "cats", "paper"];
```

Comment:
In order to complete the 404 change, I would instead search for topics that are currently in the database (using the handy model you've already made 👀 ) and then if it doesn't exist there, it's a Promise.reject! :D